docs/guides/data-movement/readme.md

```diff
@@ -90,12 +88,19 @@ The `CreateRequest` API call that is used to create Data Movement with the Copy
 options to allow a user to specify some options for that particular Data Movement. These settings
 are on a per-request basis.
 
-The Copy Offload API requires the `nnf-dm` daemon to be running on the compute node. This daemon may be configured to run full-time, or it may be left in a disabled state if the WLM is expected to run it only when a user requests it. See [Compute Daemons](../compute-daemons/readme.md) for the systemd service configuration of the daemon. See `RequiredDaemons` in [Directive Breakdown](../directive-breakdown/readme.md) for a description of how the user may request the daemon, in the case where the WLM will run it only on demand.
+The Copy Offload API requires the `nnf-dm` daemon to be running on the compute node. This daemon may
+be configured to run full-time, or it may be left in a disabled state if the WLM is expected to run
+it only when a user requests it. See [Compute Daemons](../compute-daemons/readme.md) for the systemd
+service configuration of the daemon. See `RequiredDaemons` in [Directive
+Breakdown](../directive-breakdown/readme.md) for a description of how the user may request the
+daemon, in the case where the WLM will run it only on demand.
 
-If the WLM is running the `nnf-dm` daemon only on demand, then the user can request that the daemon be running for their job by specifying `requires=copy-offload` in their `DW` directive. The following is an example:
+If the WLM is running the `nnf-dm` daemon only on demand, then the user can request that the daemon
```

Comment:
Strike this paragraph. In the paragraph above you've already told them where to look for this info.

I'm not so keen on maintaining the same info in 3 different docs.